extensions/who-is-visiting-this-content/www/styles.css

```diff
@@ -0,0 +1,52 @@
+/* Styles used by the Most Used Content table */
+.metrics-tbl {
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+}
+
+.metrics-tbl a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.metrics-tbl a:hover,
+.metrics-tbl a:focus {
+  text-decoration: underline;
+  text-decoration-thickness: max(1px, 0.0625rem);
+}
+
+/* Styles for the column headers */
+.header {
+  border-bottom: 2px solid #555;
+  font-size: 0.8125rem;
+  font-weight: 400;
+  text-transform: uppercase;
+}
```

Comment:
Are these being used? Maybe I was looking at the wrong column headers. They seem to use `rt-sort-header`

Reply:
All of your CSS feedback is much appreciated!

Reply:
Interesting. They definitely aren't active here, but I was following this example from the reactable site: https://glin.github.io/reactable/articles/twitter-followers/twitter-followers.html. I think the default formatting is nicer than what this was going for though (it was specifically replicating FiveThirtyEight's style) so I'll delete this.

Reply:
Oh LOL the reactable example manually sets the `headerClass` argument.